dotnet/src/AutoGen.Core/GroupChat/Graph.cs

```diff
@@ -12,9 +12,16 @@ public class Graph
 {
     private readonly List<Transition> transitions = new List<Transition>();
 
-    public Graph(IEnumerable<Transition> transitions)
+    public Graph()
     {
-        this.transitions.AddRange(transitions);
+    }
+
+    public Graph(IEnumerable<Transition>? transitions)
```

Comment:
Why `transitions` can be null in this case. Would it be better to simply does a null check and throw an exception when `transitions` is null in this constructor?

Reply:
In some cases the external developer may pass in a null List<Transition> Object. This is allowed at least at compile time, so it's a risk. Because in some cases you don't need to set up the transition when creating Graph entity, but dynamically configure it later, there is no need to create a new object in this case.